ip-messaging/rest/credentials/retrieve-credentials/retrieve-credentials.4.x.rb

```diff
@@ -1,11 +0,0 @@
-require "http"
-require 'twilio-ruby'
-
-# Initialize the client
-account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
-auth_token = 'your_auth_token'
-ip_messaging_client = Twilio::REST::IpMessagingClient.new(account_sid, auth_token)
-
-# Retrieve the credential
-credential = ip_messaging_client.credentials.get('CRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
-puts credential
```

Comment:
There is a bug in `twilio-ruby` v4 affecting `credentials` and this version of the library is not maintained anymore.